gradle.properties

```diff
@@ -11,8 +11,8 @@
 
 org.gradle.warning.mode=none
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx3g -XX:+HeapDumpOnOutOfMemoryError -Xss2m
-options.forkOptions.memoryMaximumSize=2g
+org.gradle.caching=true
+org.gradle.jvmargs=-Xms3g -XX:+HeapDumpOnOutOfMemoryError -Xss2m
```

Comment:
cache setting looks fine, but i'm concerned with the other change. Setting minimum heap requirement so high can cause gradle daemon to not start on normal dev machines which quite easily may have less than 3gb free memory available. Also removing max heap limit is not a good idea i think.